plugin/mmdb/setup.go

```diff
@@ -0,0 +1,94 @@
+package mmdb
+
+import (
+	"net"
+	"path/filepath"
+
+	"github.com/coredns/caddy"
+	"github.com/coredns/coredns/core/dnsserver"
+	"github.com/coredns/coredns/plugin"
+	"github.com/oschwald/geoip2-golang"
+)
+
+const pluginName = "mmdb"
+
+func init() { plugin.Register(pluginName, setup) }
+
+func setup(c *caddy.Controller) error {
+	mmdb, err := mmdbParse(c)
+	if err != nil {
+		return plugin.Error(pluginName, err)
+	}
+
+	dnsserver.GetConfig(c).AddPlugin(func(next plugin.Handler) plugin.Handler {
+		mmdb.Next = next
+		return mmdb
+	})
+
+	return nil
+}
+
+// probingIP is an arbitrary internet IP used to discover database types through database lookups.
+var probingIP = net.ParseIP("81.2.69.142")
```

Comment:
use something from rfc 1918 if possible, otherwise the IPv4 documenation IP address

Reply:
I guess you mean to use a private IP instead? of course that's no problem at all.

> otherwise the IPv4 documenation IP address

I didn't get that, sorry.